include/tvm/relay/builder.h

```diff
@@ -0,0 +1,69 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file builder.h
+ * \brief A helper file that make building expr easier.
+ */
```

Comment:
Move these content to https://github.com/dmlc/tvm/blob/master/src/relay/pass/pattern_util.h, for now let us minimize the include files